pkg/services/store/service.go

```diff
@@ -117,48 +120,43 @@ func (s *standardStorageService) Read(ctx context.Context, user *models.SignedIn
 	return s.tree.GetFile(ctx, getOrgId(user), path)
 }
 
-func isFileTypeValid(filetype string) bool {
-	if (filetype == "image/jpeg") || (filetype == "image/jpg") || (filetype == "image/gif") || (filetype == "image/png") || (filetype == "image/webp") {
-		return true
-	}
-	return false
-}
-
 type UploadRequest struct {
 	Contents           []byte
-	MimeType           string
+	MimeType           string // TODO: remove MimeType from the struct once we can infer it from file contents
 	Path               string
 	CacheControl       string
 	ContentDisposition string
 	Properties         map[string]string
```

Comment:
Perhaps unrelated to this PR... but why are these part of the upload request?

Reply:
that's a KV map that could be used by the client for storing any additional information about the file. for sql-backed storage contents of that map would land in `file_meta`

Reply:
yes -- We will need to evaluate where in the stack of abstractions this should be exposed, but yes makes sense